app/controllers/civil_servants/services_controller.rb

```diff
@@ -24,5 +31,18 @@ def load_filters
         show_all: params.dig(:filters, :show_all) == 'true'
       }
     end
+
+    def render_pdf
+      respond_with_pdf Pdfs::ServiceAgreement::FormFiller.render(
+        Pdfs::ServiceAgreement::DataProvider.evaluate_data_hash(
+          Pdfs::ServiceAgreement::GermanFormFields::SERVICE_AGREEMENT_FIELDS,
+          service: @service
+        ),
+        Rails.root.join('lib/assets/pdfs/german_service_agreement_form.pdf').to_s
+      ), I18n.t('pdfs.service_agreement.file_name',
```

Comment:
I'd prefer to extract this to another method for better readability
```suggestion
   def render_pdf
     Pdfs::ServiceAgreement::FormFiller.render(
        Pdfs::ServiceAgreement::DataProvider.evaluate_data_hash(
          Pdfs::ServiceAgreement::GermanFormFields::SERVICE_AGREEMENT_FIELDS,
          service: @service
        ),
        Rails.root.join('lib/assets/pdfs/german_service_agreement_form.pdf').to_s
      )
   end

    def respond_to_pdf
      respond_with_pdf render_pdf, I18n.t('pdfs.service_agreement.file_name',
```